about.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e8963cdebe.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>Nairobi Classical Festival</title>
+</head>
+<body>
+    <main>
+        <header>
+            <div class="desktop-only top-header">
+                <i class="fa fa-facebook" aria-hidden="true"></i>
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+                <p>Translate</p>
+                <p>Logout</p>
+            </div>
+    
+            <div class="desktop-only bottom-header">
+                <div class="project-logo">
+                    <a href="index.html"><img src="images/main-logo.svg" alt="logo-image"></a>
+                </div>
+                <div class="desktop-header-content">
+                    <p class="about-header">About</p>
+                    <p>News Letter</p>
+                    <p>Concert Program</p>
+                    <p class="attend">Attend</p>
+                </div>
+            </div>
+
+            <div class="top-menu">
+                <img src="images/menu.png" alt="">
+            </div>
+            <div class="header-content">
+                <p class="hello">"Hello! Musical World!"</p>
+                <h1>Nairobi Classical Concert 2022</h1>
+                <p class="intro-text2">The Concert brings together people in music, from child prodigies to
+                    classical music lovers. Spreading the classics to Africa is our passion. And we will be having
+                    lots of african prodigies perfoming  </p>
+                <p class="send-email">Please send any queries or questions regarding the concert to <span>Nairobiclassics@plan.org</span></p>
+            </div>
+        </header>
+
+        <!-- Mobile menu  -->
+
+        <div class="mobile-menu">
```

Comment:
- [ ] kindly use section tag instead of div for the semantic purpose for the entire project.

Reply:
For this part, adding a section tag will require h1-h2 tags. Since that does not apply, i reverted back to a div.